mdoc/src/main/scala-3/mdoc/internal/markdown/MarkdownCompiler.scala

```diff
@@ -166,7 +166,7 @@ class MarkdownCompiler(
         case (Right(start), Right(end)) =>
           Position.Range(start.input, start.start, end.end).toUnslicedPosition
```

Comment:
`val position = pos.get` <- never used (not related to this exact line)

Reply:
Right! Fixed together with the previous one.